truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/nodes/LoopNode.java

```diff
@@ -107,6 +110,8 @@
  * @since 0.8 or earlier
  */
 public abstract class LoopNode extends Node {
```

Comment:
Why is TVMCI in LoopNode? Having TMVCI#initializeCallTarget  in LoopNode does not feel right. Couldn't we introduce a abstract base class for TruffleRuntime instead and do the same thing with TVMCI there?